front/src/modules/trainschedule/components/Timetable/Timetable.tsx

```diff
@@ -122,7 +122,9 @@ const Timetable = ({
       return show;
     });
   }, [currentDepartureDates]);
-
+  const selectPathProjection = async (itemId: TimetableItemId) => {
+    dispatch(updateTrainIdUsedForProjection(itemId));
+  };
```

Comment:
Hum I'm not sure you should put it here, can you move it in PacedTrainItem ?

Reply:
done